OKTA-438747 Custom error guide multi-to-single page (#2740)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2018/index.md

#### Custom Okta-hosted Sign-In Page is in Early Access

> You can customize the text and the look and feel of the Okta-hosted sign-in page using form controls and an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted error page](/docs/guides/custom-error-pages/), this feature offers a fully customized end-user sign-in experience hosted by Okta. For details, see [Configure a custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/).

Reviewer: ```suggestion
You can customize the text and the look and feel of the Okta-hosted sign-in page by using form controls and an embedded HTML editor. When this feature is used with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted error page](/docs/guides/custom-error-pages/), it offers a fully customized end-user sign-in experience that is hosted by Okta. See [Configure a custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/) for more information.
```

#### Custom Error Page is in Early Access

> You can customize the text and the look and feel of error pages using an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/), this feature offers a fully customized error page. For details, see [Configure a custom error page](/docs/guides/custom-error-pages/).

Reviewer: ```suggestion
You can customize the text and the look and feel of error pages by using an embedded HTML editor. When the feature is used with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/), it offers a fully customized error page. See [Configure a custom error page](/docs/guides/custom-error-pages/) for more information.
```
